pages/ps/ps1/data.tsx

```diff
@@ -23,7 +23,7 @@ export default function PS1Data() {
     const updateData = async () => {
         setIsLoading(true);
         if (yearRef == cachedYear) return
-        const res = await fetch("/api/ps/ps2Cutoffs", {
+        const res = await fetch("/api/ps/cutoffs/ps1Cutoffs", {
```

Comment:
Can change it to `/api/ps/cutoffs/ps1` or even `/api/ps/ps1/cutoffs`, although I feel the latter is more _idiomatic_.